rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,27 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
+    selection:
+        - CommandLine|contains|all:
+          - 'mv '
+          - '/usr/local/share/ca-certificates'
+    selection2:
+        - ProcessName|contains:
+          - 'update-ca-certificates'
+    selection3:
+        - CommandLine|contains|all:
+          - 'cp '
+          - 'rootCA.crt'
+          - 'update-ca-trust'
```

Comment:
section3 has the same issue as was mentioned [here](https://github.com/Neo23x0/sigma/pull/1044#discussion_r502049487). but again, you can remove `cp` as there are many ways to move/copy/create a file in a specific directory,  and remove `rootCA.crt` as it could be any other name. 
having process name `update-ca-trust` the same way as was mentioned [here](https://github.com/Neo23x0/sigma/pull/1044/files#r519252498) is sufficient. 

Thanks!

Reply:
Thank you very much!